### DISCUSSION TITLE
[Feature] app-dedicated-tv-appletv-android

### DISCUSSION THREAD

**Discussion by emaxcs** [immich-app/immich#13748] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It’s possible to create dedicated app  for Apple TV And Android TV to show and navigate to server with the best UI. (Same like app photos Google or Apple in this specific OS TV)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#7030/#9267

**Comment by emaxcs** [comment]
@bo0tzz 

Chromecast and airplay are not the same thing as the dedicated app. With Chromecast and AirPlay, streaming to the TV creates a dependency on the smartphone and its application. With a dedicated TV application, this dependency disappears and you gain freedom of use, with a dedicated TV account and slideshow mode, so you can use your TV as an independent digital photo frame.

Do you plan to create this application and review the demand?

**Reply by bo0tzz** [reply]
Then it's a dupe of #3892.

**Comment by emaxcs** [comment]
@bo0tzz 

This topic is closed, but nothing is planned, or at least I don't see a clear answer to this request for Apple TV? Please could you clarify whether or not this is planned and for when or for what reason it is not retained?

And what about the application dedicated to Android TV, is there anything in the pipeline?

**Reply by bo0tzz** [reply]
Android TV: #1614

We have no plans for official support for either of these.

**Reply by emaxcs** [reply]
@bo0tzz thank you for your reply and for immich
